cmd/stdiscosrv/etc/linux-systemd/discosrv.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Syncthing Discovery Server
+After=network.target
+Documentation=man:stdiscosrv(1)
+
+[Service]
+WorkingDirectory=/var/lib/syncthing-stdiscosrv
```

Comment:
What's this?

Reply:
This is the directory which it will work in

Reply:
Well, that's what the name is telling me - what's it required for?

Reply:
Well no program should work in / in case it needs to put files in it's current directory

Reply:
I'd say no program should put files in it's current directory unless that's its purpose. So this setting seems superfluous to me, while not harmful. My preference is to only use options/config when necessary and again, as it's just preference, I can live with it.